src/main/java/uk/nhs/hee/tis/trainee/sync/model/Person.java

```diff
@@ -25,7 +25,7 @@
 import org.springframework.stereotype.Component;
 
 @Component("Person")
-@Scope("Prototype")
+@Scope("prototype")
```

Comment:
Good spot. `@Scope(BeanDefinition.SCOPE_PROTOTYPE))` might be useful as a bit more declarative.

Reply:
Good shout, I've switched everything over.